_posts/en/newsletters/2020-01-22-newsletter.md

```diff
@@ -0,0 +1,117 @@
+lang: en
+---
+This week's newsletter requests review of a method for sending payments
+as part of a chaumian coinjoin mix, links to a work-in-progress protocol
+specification for discreet log contracts, and includes our regular
+sections about notable changes to popular services, client software, and
+infrastructure projects.
+
+## Action items
+
+- **Help test LND 0.9.0-beta-rc3:** this [pre-release][lnd 0.9.0-beta]
+  of the next major version of LND brings several new features and bug
+  fixes.  Experienced users are encouraged to help test the software so
+  that any problems can be identified and fixed prior to release.
+
+## News
+
+- **New coinjoin mixing technique proposed:** Max Hillebrand started a
+  [thread][wormhole thread] on the Bitcoin-Dev mailing list about
+  *Wormhole*, a method developed during a [Wasabi design discussion][]
+  for sending payments as part of a chaumian coinjoin.  The protocol
+  prevents even the spender from learning the receiver's Bitcoin address
+  (within the limits of the anonymity set).  Developer ZmnSCPxj
+  [notes][zmn note] that the technique is similar to [tumblebit][], which
+  provides a trustless chaumian payment service.  Hillebrand is
+  requesting feedback on the design in the hopes of seeing it
+  implemented in the future.
+
+- **Protocol specification for discreet log contracts (DLCs):** [DLCs][] are
+  a contract protocol where two or more parties agree to a exchange
+  money dependent on the outcome of a certain event as determined by an
+  oracle (or several oracles).  After the event happens, the oracle
+  publishes a commitment to the outcome of the event in the form of a
+  digital signature, which the winning party can use to claim their
+  funds.  The oracle doesn't need to know the terms of the contract (or
+  even that there is a contract).  The contract can look like either the
+  onchain part of an LN transaction or it can be executed within an LN
+  channel.  This makes DLCs more private and efficient than other known
+  oracle-based contract methods, and it's arguably more secure as an
+  oracle who commits to a false result generates clear evidence of their
+  fraud.
+
+    This week Chris Stewart [announced][stewart dlc] that several
+    developers are working on a specification for using DLCs, with the
+    goal of creating a interoperable design for use between different
+    software, including LN implementations.  See their
+    [repository][dlcspecs] for the current specification.  Anyone
+    interested in DLCs may also wish to review the [scriptless
+    scripts][scriptless scripts examples] repository that documents
+    other clever applications of digital signature schemes to contract
+    protocols.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **River Financial launches using variety of Bitcoin tech:** In a [Twitter thread][river twitter thread]
```

Comment:
By focusing on the launch aspect of this, I think we could leave ourselves open to being accused of promoting a member's service. I think it's fine to cover this item if we just tweak the wording a bit so it's solely focused on the tech (and perhaps only link to Glazman's thread rather than the launch blog post).

Reply:
Tweaked the language here, thanks @jnewbery for the comment